LibertyDSNP/sdk-ts
# Features/npm package 177120105

Problem
=======
On creation of a GitHub release, an action should trigger the publishing of a new version of the npm package
[#177120105](https://www.pivotaltracker.com/story/show/177120105)

Solution
========

Cleanup the built files and Add a GitHub action for publishing

Change summary:
---------------

* Switch announcement to use index.ts and update the exporter with the announcement module
* Update the package org to @unfinishedlabs
* Update includes
* Add quick start docs
* Switch publishing the docs to release to sync with package publishing
* Add a release --dry-run to the main build for safety
* Add a package release action
* Release a @next package after each merge to main.


Steps to Verify:
----------------
1. Create our first release (v0.0.1)
1. See that the package is published to npm!
1. See that the docs are published on github pages
